marconi-cardano-indexers/test-lib/Test/Gen/Marconi/Cardano/Indexers/UtxoQuery.hs

```diff
@@ -0,0 +1,136 @@
+{-# LANGUAGE FlexibleContexts #-}
```

Comment:
Noting that this module is entirely extracted from `Test.Spec.Marconi.Cardano.Indexers.UtxoQuery` . Unlike with the others, I did not add anything here.